package.json

```diff
@@ -32,17 +32,22 @@
   "license": "MIT",
   "homepage": "https://github.com/humanprotocol/eth-kvstore",
   "dependencies": {
+    "@truffle/hdwallet-provider": "^2.0.17",
     "armlet": "^0.1.8",
     "dotenv": "^6.0.0",
     "truffle": "^5.1.32",
     "truffle-hdwallet-provider": "^1.0.6"
```

Comment:
can we remove this now?

Reply:
@spiritbrother you removed `@truffle/hdwallet-provider` I think @posix4e meant to remove `truffle-hdwallet-provider`, since this package has been moved to `@truffle/hdwallet-provider`
Have in mind that if you this you need to find references to `truffle-hdwallet-provider` and replace them by `@truffle/hdwallet-provider`
![image](https://user-images.githubusercontent.com/61605646/197174732-4c40ab66-3e8f-4146-97e1-794a1b8f2ac9.png)